schemas/observer.yml

```
@@ -90,3 +90,41 @@
         There is no predefined list of observer types. Some examples are
         `forwarder`, `firewall`, `ids`, `ips`, `proxy`, `poller`, `sensor`, `APM server`.
       example: firewall
+
+    - name: ingress
+      level: extended
+      type: object
+      short: Object field for ingress information
+      description: >
+        Observer.ingress holds information like interface number and name, vlan, and zone information to 
+        classify ingress traffic.  Single armed monitoring such as a network sensor on a span port should 
+        only use observer.ingress to categorize traffic.
+
+    - name: ingress.zone
+      level: extended
+      type: keyword
+      short: Observer ingress Zone
+      description: >
+        Network zone of incoming traffic as reported by the observer to categorize the source area of ingress 
+        traffic.
+      
+      example: Internal, External, DMZ, HR, Legal, etc.
+
+    - name: egress
+      level: extended
+      type: object
+      short: Object field for egress information
+      description: >
+        Observer.egress holds information like interface number and name, vlan, and zone information to 
+        classify egress traffic.  Single armed monitoring such as a network sensor on a span port should 
+        ohly use observer.ingress to categorize traffic.
+
+    - name: egress.zone
+      level: extended
+      type: keyword
+      short: Observer Egress Zone
+      description: >
+        Network zone of incoming traffic as reported by the observer to categorize the destination area of egress 
```

```suggestion
        Network zone of outgoing traffic as reported by the observer to categorize the destination area of egress 
```

Should "incoming" be replaced by "outgoing", here?

---

Fixed